src/datatypes/src/schema/column_schema.rs

```diff
@@ -773,12 +773,17 @@ pub struct SkippingIndexOptions {
     /// The granularity of the skip index.
     pub granularity: u32,
     /// The false positive rate of the skip index (in ten-thousandths, e.g., 100 = 1%).
+    #[serde(default = "skipping_index_options_default_false_positive_rate_in_10000")]
     pub false_positive_rate_in_10000: u32,
     /// The type of the skip index.
     #[serde(default)]
     pub index_type: SkippingIndexType,
 }
 
+fn skipping_index_options_default_false_positive_rate_in_10000() -> u32 {
```

Comment:
Can we reused the [`fulltext_options_default_false_positive_rate_in_10000`](https://github.com/GreptimeTeam/greptimedb/blob/599f289f59b347aecb271ce4f2c0d7f217e4847e/src/datatypes/src/schema/column_schema.rs#L538) and rename it to `default_false_positive_rate_in_10000` that can be shared between `FulltextOptions` and `SkippingIndexOptions`?